test/bin/ci_phase_iso_build.sh

```diff
@@ -130,16 +130,13 @@ $(dry_run) bash -x ./bin/build_rpms.sh
 # Set up for scenario tests
 $(dry_run) bash -x ./bin/create_local_repo.sh
 
-# Start the web server to host the ostree commit repository for parent images
-$(dry_run) bash -x ./bin/start_webserver.sh
-
 if ${COMPOSER_CLI_BUILDS} ; then
     # Figure out an optimal number of osbuild workers
     CPU_CORES="$(grep -c ^processor /proc/cpuinfo)"
     MAX_WORKERS=$(find "${ROOTDIR}/test/image-blueprints" -name \*.toml | wc -l)
     CUR_WORKERS="$( [ "${CPU_CORES}" -lt  $(( MAX_WORKERS * 2 )) ] && echo $(( CPU_CORES / 2 )) || echo "${MAX_WORKERS}" )"
 
-    $(dry_run) bash -x ./bin/start_osbuild_workers.sh "${CUR_WORKERS}"
+    $(dry_run) bash -x ./bin/manage_composer_config.sh create-workers "${CUR_WORKERS}"
```

Comment:
Let's move the core math to the `manage_composer_config.sh` script and get rid of the `<num>` argument?

Reply:
Done, I made it so when running `manage_composer_config.sh create-workers`, it determines the ideal number of workers, but I left the option to create a specific number of workers by running `manage_composer_config.sh create-workers [number]`.